### PR TITLE
Update translations from Lionbridge for 4.5

### PR DESCRIPTION
**Pull request by Ghostkeeper**
These are the new translations that we got from the translation service.

Contributes to issue CURA-7201.

### CODE REVIEW

File: resources/i18n/de_DE/cura.po [hunk 1]
```diff
@@ -200,24 +200,24 @@ msgstr "Plugin für Lizenzvereinbarung"
...
 "\n"
 "Syncing..."
-msgstr ""
+msgstr "\nSynchronisierung läuft ..."
 
 #: /home/ruben/Projects/Cura/plugins/Toolbox/src/CloudSync/RestartApplicationPresenter.py:18
 msgctxt "@info:generic"
 msgid "You need to quit and restart {} before changes have effect."
-msgstr ""
+msgstr "Sie müssen das Programm beenden und neu starten {{}, bevor Änderungen wirksam werden."
```
Comment (Vandrasc) [inline resources/i18n/de_DE/cura.po]: Extra bracket is unnecessary ...

File: resources/i18n/nl_NL/cura.po [hunk 2]
```diff
@@ -168,23 +168,23 @@ msgctxt "@info:generic"
 msgid ""
 "\n"
 "Do you want to sync material and software packages with your account?"
-msgstr ""
+msgstr "\nWilt u materiaal- en softwarepackages synchroniseren met uw account?"
```
Comment (Vandrasc) [inline resources/i18n/nl_NL/cura.po]: This doesn't sound very correct to me.

File: resources/i18n/nl_NL/cura.po [hunk 3]
```diff
@@ -699,7 +699,7 @@ msgstr "X3D-bestand"
 #: /home/ruben/Projects/Cura/plugins/SimulationView/SimulationView.py:119
 msgctxt "@info:status"
 msgid "Cura does not accurately display layers when Wire Printing is enabled."
-msgstr ""
+msgstr "Als Draadprinten is ingeschakeld, geeft Cura lagen niet goed weer"
```
Comment (Vandrasc) [inline resources/i18n/nl_NL/cura.po]: This sounds bad as well.

File: resources/i18n/de_DE/cura.po (hunk 1 above)
Comment (Ghostkeeper) [reply resources/i18n/de_DE/cura.po]: Fixed this. Thanks.

**Pull request merged**